jxlint/src/main/java/com/selesse/jxlint/model/ProgramOptions.java

```diff
@@ -69,6 +71,32 @@ public OutputType getOutputType() {
         }
     }
 
+    public String getSourcePathPrefix() {
+        String result;
+        if(options.containsKey(JxlintOption.SRC_PATH_PREFIX)) {
+            result = options.get(JxlintOption.SRC_PATH_PREFIX);
+        } else {
+            if(getSourceDirectory() == null) {
+                return "";
+            }
+            if (!options.containsKey(JxlintOption.WEB) && options.containsKey(JxlintOption.OUTPUT_TYPE_PATH)) {
```

Comment:
Yeah, I don't know how we could get this to work for `localhost:8380/report.jsp`. Maybe it should be unsupported.